README.md

```diff
@@ -34,81 +37,126 @@
+- An easy-to-use trainer that has been written to be as performant as possible and [integrates best practices](https://www.mosaicml.com/blog/5-best-practices-for-efficient-model-training) for efficient training.
+- Functional forms of all of our speedup methods that allow you to integrate them into your existing training loop.
+- Strong, _reproducible_ baselines to get you started as quickly as possible.
+
+### Benefits
+
+Composer offers significant reductions in training time and cost. Using Composer, you can:
+
+- Train ResNet-50 on ImageNet to the standard 76.6% top-one accuracy for $43 in 1 hour and 20 minutes (_with vanilla PyTorch:_ $127 in 3 hours and 53 minutes) on AWS.
+- Train a GPT-2 125M to a standard perplexity of 23.9 for $142 in 4 hours and 34 minutes (_with vanilla PyTorch_: $255 in 7 hours and 47 minutes) on AWS.
+- Use these speedup methods on your own models in your own training loop.
 
-- Train an ImageNet model to 76.1% accuracy for $37 (_with vanilla PyTorch:_ $127)
-- Train a GPT-2 125M to a perplexity of 23.9 for $148 (_with vanilla PyTorch_: $255)
-- Use start-of-the-art implementations of methods to speed up your own training loop.
+### The Latest Research on Efficient Training
 
-At MosaicML, we are focused on making training ML models accessible. To do this, we continually productionize state-of-the-art academic research on efficient model training, and also study the _combinations_ of these methods in order to ensure that model training is ✨ as efficient as possible ✨.
+At MosaicML, our goal is to make training state-of-the-art deep learning models accessible to everyone.
+We do so through Composer:
 
-Everyone has their own priorities: best accuracy, cheapest cost, and somewhere in between.  Composer provides novel recipes that push the boundary of both cost and accuracy. Composer allows you to choose **the best model for your real-world constraints**.
+* We continually integrate <b>state-of-the-art research on efficient neural network training</b> (and our improvements to that research).
+* These speedup methods work on our <b>efficient implementations of up-to-date models and datasets</b>.
+* We provide <b>curated recipes of speedup methods</b> that lead to the most efficient possible training on these benchmarks.
 
-Composer features:
-- 20+ efficient training methods for training a better language and vision models! Don't waste hours trying to reproduce research papers when Composer has done the work for you.
-- Easy-to-use (optional) Trainer interface written to be as performant as possible, and [integrated best practices](https://www.mosaicml.com/blog/5-best-practices-for-efficient-model-training).
-- Easy-to-use Functional forms that allow you to integrate efficient training methods into your training loop!
-- Strong, _reproducible_ baselines to get you started as 💨 fast 💨 as possible
 
-## Quickstart
+## 📣 Quickstart
```

Comment:
Can we move the quickstart (or at least the installation section) up higher? I think it would be helpful if it is above the fold